react-hot-loader/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-hot-loader
+// Project: https://github.com/gaearon/react-hot-loader
+// Definitions by: Jacek Jagiello <https://github.com/jacekjagiello/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react"
+
+declare module "react-hot-loader" {
+
+  interface ErrorReporterProps {
+    error: any
+  }
+
+  export interface AppContainerProps {
+    children?: React.ReactElement<any>,
+    errorReporter?: React.StatelessComponent<ErrorReporterProps>
```

Comment:
The `errorReporter` type can also be a `React.ComponentClass`, or basically anything you can give as the first argument to `React.createElement` besides `string`.

Though it appears to typecheck even with a class component?